common/user-auth/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "user-auth-common"
+version = "0.1.0"
```

Comment:
nit: I like leaving this at `0.0.0` until we actually make releases, where it actually matters